focus/internals/src/lib/model/selection/project.rs

```diff
@@ -60,32 +66,37 @@ impl Project {
             }
 
             for target in &project.targets {
-                let (prefix, rest) = match target.split_once(':') {
-                    Some((prefix, rest)) => (prefix, rest),
+                target_set.insert(Target::try_from(target.as_str())?);
+            }
+
+            for project in &project.projects {
+                match available_subprojects.get(project) {
+                    Some(subproject) => {
+                        resolvable_projects.push(subproject);
+                    }
                     None => {
-                        bail!(TargetError::NoSchemeProvidedError);
+                        bail!("Invalid project target: {}", project.as_str())
                     }
                 };
-                let rest = rest.to_owned();
-                if prefix.eq_ignore_ascii_case("project") {
-                    match available_subprojects.get(&rest) {
-                        Some(subproject) => {
-                            resolvable_projects.push(subproject);
-                        }
-                        None => {
-                            bail!("Invalid project target: {}", target.as_str())
-                        }
-                    };
-                } else {
-                    target_set.insert(Target::try_from(target.as_str())?);
-                }
             }
         }
 
         debug!(seen_projects = ?seen_projects, "Saw these projects while resolving top-level project");
 
         Ok(target_set)
     }
+
+    pub fn lint(&self) -> Result<()> {
```

Comment:
This current iteration of lint only validates targets. Would it be a good idea to add project linting next?

Reply:
> Really like what you did with the projects.
> 
> When can the change to project definitions change to include this extra field? If we were to ship these changes will there be any issues with reading the current project definitions, or vice versa: change the project definitions and ship the changes after?

I found that `projects:` were not being used in source, so not addressing this. Although we will want to make sure this is still true before release. I'll add this to the release ticket.